responder/jira/jira.go

```diff
@@ -0,0 +1,186 @@
+		},
+	}
+
+	if opts.Priority != "" {
+		i.Fields.Priority = &jira.Priority{Name: opts.Priority}
+	}
+
+	issue, _, err := jc.client.Issue.Create(&i)
+	if err != nil {
+		return nil, err
+	}
+	logger.Debugf("[Jira] Issue created for Project: [%s] with ID: [%s] - [%s]", opts.ProjectKey, issue.ID, opts.Summary)
+
+	return issue, nil
+}
+
+func (jc JiraClient) AddComment(issueID, comment string) (*jira.Comment, error) {
+
+	c, _, err := jc.client.Issue.AddComment(issueID, &jira.Comment{Body: comment})
+	if err != nil {
+		return nil, err
+	}
+
+	logger.Debugf("[Jira] Comment: [%s] added for issueID: [%s]", c.Body, issueID)
+
+	return c, nil
+}
+
+func (jc JiraClient) GetComments(issueID string) ([]api.Comment, error) {
+
+	issue, _, err := jc.client.Issue.Get(issueID, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	var comments []api.Comment
+	for _, comment := range issue.Fields.Comments.Comments {
+		createdAt, _ := time.Parse("2006-01-02T15:04:05.999-0700", comment.Created)
+		comments = append(comments, api.Comment{
+			Body:      comment.Body,
+			CreatedBy: comment.Author.DisplayName,
+			CreatedAt: createdAt,
+		})
+	}
+
+	return comments, nil
+}
+
+func (jc JiraClient) GetConfig() (map[string]JiraProject, error) {
+	priorities, _, err := jc.client.Priority.GetList()
+	if err != nil {
+		return nil, err
+	}
+
+	var priorityList []string
+	for _, priority := range priorities {
+		priorityList = append(priorityList, priority.Name)
+	}
+
+	statuses, _, err := jc.client.Status.GetAllStatuses()
+	if err != nil {
+		return nil, err
+	}
+	var statusList []string
+	for _, status := range statuses {
+		statusList = append(statusList, status.Name)
+	}
+
+	projects, _, err := jc.client.Project.GetList()
+	if err != nil {
+		return nil, err
+	}
+
+	p := make(map[string]JiraProject)
+	for _, projectMeta := range *projects {
+		project, _, err := jc.client.Project.Get(projectMeta.ID)
+		if err != nil {
+			return nil, err
+		}
+
+		var issueTypes []string
+		for _, issueType := range project.IssueTypes {
+			issueTypes = append(issueTypes, issueType.Name)
+		}
+
+		p[projectMeta.Name] = JiraProject{
+			Key:        project.Key,
+			Name:       project.Name,
+			IssueTypes: issueTypes,
+			Priorities: priorityList,
+			Statuses:   statusList,
+		}
+	}
+
+	return p, nil
+}
+
+func (jc JiraClient) GetIssueTransitions(issueID string) ([]JiraIssueTransitions, error) {
+	transitions, _, err := jc.client.Issue.GetTransitions(issueID)
+	if err != nil {
+		return nil, err
+	}
+
+	var transitionList []JiraIssueTransitions
+	for _, transition := range transitions {
+		transitionList = append(transitionList, JiraIssueTransitions{
+			ID:      transition.ID,
+			Name:    transition.Name,
+			ToState: transition.To.Name,
+		})
+	}
+	return transitionList, err
+}
+
+func (jc JiraClient) TransitionIssue(issueID, transitionID string) error {
+	_, err := jc.client.Issue.DoTransition(issueID, transitionID)
+	if err != nil {
+		return err
+	}
+	return nil
```

Comment:
```suggestion
	return err
```

Reply:
umm, this should be `nil` right ? cause if `err` we are returning that in the previous line